test/integration/hmrc_manual_section_test.rb

```diff
@@ -12,6 +12,12 @@ class HmrcManualSectionTest < ActionDispatch::IntegrationTest
     end
   end
 
+  test "partial has no content id" do
```

Comment:
Ah, it was because I assumed the page set-up was running before each block but actually you needed to include it in a block for it navigate/render the page. This is now updated.